hi-ml-histopathology/testhisto/testhisto/utils/utils_slides_datamodule.py

```diff
@@ -0,0 +1,248 @@
+#  ------------------------------------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation. All rights reserved.
+#  Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+#  ------------------------------------------------------------------------------------------
+import os
+from pathlib import Path
+import py
+import medmnist
+import numpy as np
+import pandas as pd
+import torchvision.transforms as transforms
+
+from enum import Enum
+from torch import Tensor
+from medmnist import INFO
+from tifffile import TiffWriter
+
+from torch.utils.data import DataLoader
+from typing import Tuple, Optional, List, Union
+from histopathology.datamodules.base_module import SlidesDataModule
+from histopathology.datasets.base_dataset import SlidesDataset
+from health_azure.utils import PathOrString
+
+
+class MockSlidesDataset(SlidesDataset):
+    """Mock and child class of SlidesDataset, to be used for testing purposes.
+    It overrides the following, according to the PANDA cohort settings:
+
+    :param SLIDE_ID_COLUMN: CSV column name for slide ID set to "image_id".
+    :param LABEL_COLUMN: CSV column name for tile label set to "isup_grade".
+    :param METADATA_COLUMNS: Column names for all the metadata available on the CSV dataset file.
+    """
+
+    SLIDE_ID_COLUMN = "image_id"
```

Comment:
not entirely sure why we want to overwrite `SlidesDataset.SLIDE_ID_COLUMN`

Reply:
I overwrote it as in PandaDataset but it might be unnecessary!